### PR TITLE
handle event by mini player

### PR DESCRIPTION
**Pull request by trantan97**
Điều khiển chơi nhạc bằng mini player
https://edu-redmine.framgia.vn/issues/5707

### CODE REVIEW

**Comment by trantan97** [comment]
ready

File: app/src/main/java/com/trantan/music53/service/music/PlayService.java [hunk 1]
```diff
@@ -142,9 +202,51 @@ public void addTrack(Track track) {
...
+        public void setRunable(boolean runable) {
+            mRunable = runable;
+        }
+
+        @Override
+        protected Void doInBackground(Void... voids) {
+            while (mRunable) {
+                publishProgress(mPlayerManager.getCurrentTime());
+                try {
+                    Thread.sleep(1000);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/trantan/music53/service/music/PlayService.java]: hardcode

File: app/src/main/java/com/trantan/music53/service/music/PlayService.java [hunk 2]
```diff
@@ -142,9 +202,51 @@ public void addTrack(Track track) {
...
+        mListeners.add(listener);
+    }
+
     public class MyBinder extends Binder {
         public PlayService getService() {
             return PlayService.this;
         }
     }
+
+    class CurrentTimeTask extends AsyncTask<Void, Integer, Void> {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/trantan/music53/service/music/PlayService.java]: private static class

**Comment by trantan97** [comment]
ready

File: app/src/main/java/com/trantan/music53/service/music/PlayService.java [hunk 3]
```diff
@@ -142,9 +208,69 @@ public void addTrack(Track track) {
...
+
+        @Override
+        protected Void doInBackground(PlayService... playServices) {
+            mService = playServices[0];
+            while (mRunable) {
+                publishProgress(mService.mPlayerManager.getCurrentTime());
+                try {
+                    Thread.sleep(SLEEP_TIME);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/trantan/music53/service/music/PlayService.java]: handle exception

**Comment by trantan97** [comment]
ready

**Pull request merged**